services/frontend-service/src/ui/components/ReleaseDialog/ReleaseDialog.tsx

```diff
@@ -94,20 +107,8 @@ export const EnvironmentListItem: React.FC<EnvironmentListItemProps> = ({
                 },
             });
         }
-    }, [app, env.name, release.version]);
-    const createAppLock = useCallback(() => {
-        addAction({
-            action: {
-                $case: 'createEnvironmentApplicationLock',
-                createEnvironmentApplicationLock: {
-                    environment: env.name,
-                    application: app,
-                    lockId: '',
-                    message: '',
-                },
-            },
-        });
-    }, [app, env.name]);
+        createAppLock();
```

Comment:
I'm a bit surprised that you needed to change so few tests. Do we really not check the actions in cart after clicking a button?

Reply:
You are right. I added some `expect`s to 2 tests